project/jteng2127/README.rst

```diff
@@ -0,0 +1,199 @@
+##########
+ PyBezier
+##########
+
+*******************
+ Basic Information
+*******************
+
+**Project**: pybezier
+
+**Github url**: https://github.com/jteng2127/pybezier
+
+******************
+ Problem to Solve
+******************
+
+In the fields of Aerodynamics, Fluid mechanics, and Modeling, it's often
+essential to describe complex curves accurately. One approach involves
+the use of Bezier curves, which are widely applied in computer graphics
+because their smoothness and intuitive control.
+
+The goal is to use multiple third-order (or higher order) Bezier curves
+to fit the following types of curve:
+
+#. A fourth-order Bezier curve
+#. A NACA 4-digit airfoil [1]_
+#. Any arbitrary curve
+
+The potential solutions are listed in :ref:`potential-solutions`
+
+*********************
+ System Architecture
+*********************
+
+The algorithm will be implemented in C++ for performance and efficiency,
+with Python bindings provided via pybind11.
+
+Input
+=====
+
+The fitting target can be one of the following:
+
+-  A list of points representing the curve
```

Comment:
How many data points do you plan to process in the system?

Reply:
Around 20 to 200 data points during training stage, and use 2x to 5x data points during error calculation.

Reply:
200 are a good start.